Python/compile.c

```diff
@@ -1519,6 +1553,17 @@ compiler_addop_j(struct compiler *c, int opcode, basicblock *b, int absolute)
     } \
 }
 
+/* These macros allows to check only for errors and not emmit bytecode
+ * while visiting nodes.
+*/
+
+#define BEGIN_DO_NOT_EMIT_BYTECODE { \
```

Comment:
`BEGIN_DO_NOT_EMIT_BYTECODE` and `END_DO_NOT_EMIT_BYTECODE` always surround `VISIT_SEQ`. I would add a new macros which expands to
```
c->c_do_not_emit_bytecode++;
VISIT_SEQ(...);
c->c_do_not_emit_bytecode--;
```
instead of `BEGIN_DO_NOT_EMIT_BYTECODE` and `END_DO_NOT_EMIT_BYTECODE`.

Reply:
Will update the PR soon.